packages/connect/src/device/DeviceList.ts

```diff
@@ -434,7 +434,8 @@ export class DeviceList extends TypedEmitter<DeviceListEvents> {
                     path: d.path,
                     product: d.product,
                 };
-                this.devices[d.path].activitySessionID = d.session;
+                // TODO: is this ok? transportSession should be set only as result of acquire/release
+                // this.devices[d.path].transportSession = d.session;
```

Comment:
controversy here

Reply:
yes, I would also like to have this deleted. it was there for a reason I don't remember now but I will check on it.

Reply:
it looks ok.  I am still a bit paranoic but yes, lets get rid of it.